ldes-server-port-ingestion-rest/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+		<artifactId>ldes-server</artifactId>
+		<version>0.0.2-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>ldes-server-port-ingestion-rest</artifactId>
+
+	<dependencies>
+		<!-- ############################################################## -->
+		<!-- VSDS -->
+		<!-- ############################################################## -->
+		<dependency>
+			<groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+			<artifactId>ldes-server-domain</artifactId>
+		</dependency>
+		<!-- ############################################################## -->
+		<!-- SPRING BOOT -->
+		<!-- ############################################################## -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+        <dependency>
```

Comment:
Spacing?

Reply:
Agreed. Spacing is one of the issues on my list for the coding conventions meeting tomorrow. We should use CheckStyle.